modules/nwtc-library/src/NWTC_C_Binding.f90

```diff
@@ -28,7 +28,7 @@ MODULE NWTC_C_Binding
 !     This must exactly match the value in the python-lib. If ErrMsgLen changes at
 !     some point in the nwtc-library, this should be updated, but the logic exists
 !     to correctly handle different lengths of the strings
-INTEGER(IntKi), PARAMETER   :: ErrMsgLen_C = 1025
+INTEGER(IntKi), PARAMETER   :: ErrMsgLen_C = 8196
```

Comment:
Might need this to be 8197.  See note above https://github.com/OpenFAST/openfast/pull/2735/files#r2069267230

Reply:
Fixed